workshops/index.rst

```diff
@@ -16,21 +16,21 @@ Attending a workshop is the best way to get started **using and developing** GMT
 Upcoming Workshops
 ------------------
 
-2022-04 | Crafting beautiful maps with PyGMT at EGU General Assembly 2022
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+2022-05-24 | Crafting beautiful maps with PyGMT at EGU General Assembly 2022
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
 * **Title:** Crafting beautiful maps with PyGMT
 * **Instructors:** Wei Ji Leong, Leonardo Uieda, Meghan Jones, Andre Luiz Belem
-* **When:** Early April 2022
-* **Where:** Hybrid (Online and at Vienna, Austria)
-* **Repository:** TBA
+* **When:** 24 May 2022, 13:10-14:40 (UTC) / 15:10-16:40 (CEST)
+* **Where:** Online
+* **Repository:** https://github.com/GenericMappingTools/egu22pygmt
 * **EGU Session info:** https://meetingorganizer.copernicus.org/EGU22/session/43186
-* **Application Portal:** TBA
+* **Application Portal:** https://egu22.eu/register_and_venue/registration.html
 
 This short course is geared towards geoscientists interested in creating beautiful maps using Python.
 Only basic Python knowledge is needed, and a background in cartography is not required to use PyGMT effectively!
-The 3.5 hour long short course will be based on content adapted from https://github.com/GenericMappingTools/2021-unavco-course
-and https://github.com/GenericMappingTools/foss4g2019oceania. Each of the 30–45 minute sessions will involve a
+The 1.5 hour long short course will be based on content adapted from https://github.com/GenericMappingTools/2021-unavco-course
+and https://github.com/GenericMappingTools/foss4g2019oceania. Each of the 30 minute sessions will involve a
 quick (~10 minute) walkthrough by the speaker, followed by a more hands-on session in breakout rooms where tutorial
```

Comment:
Cc @leouieda, @meghanrjones, @andrebelem. Since our EGU22 short course duration is now 1hr 30min (90mins), does it make sense to have 3x 30min chunks? Or a 10min intro + 4x 20min sessions? What do you all think?

Reply:
Sorry for my delay here ! @weiji14 I like the format 10+4x20. Last time I made something similar in python it worked well. It means we can work on blocks with enought time to show several application cases.
What do you think ? and @leouieda and @meghanrjones ?

Reply:
10 min intro + 4x20 min sessions sounds good to me, though I think we could adjust this as needed when creating the content. Should we schedule a zoom or gather.town get-together on our own to include in this website announcement? It doesn't sound like the EGU virtual only events will have any automatic interactive component.

Reply:
Ok, looks like 10+4x20 has 3 points, but we'll still need to check with @leouieda if he's ok with doing the 1 out of 4 :wave:

An after-workshop get-together sounds good, I didn't attend vEGU21 last year, but I heard there was a gather.town. Could we just use that instead? Not quite sure how it would work to be honest.

Reply:
Sorry, missed this thread completely. Are we doing separate recordings and then merging them for the upload? I can do 20 min, no problem. What’s the deadline for this?

Reply:
Yep, we'll probably do separate recordings and merge them together. I've decided we should have an individual deadline on Friday 6 May 2022 to upload the recording, see the thread at https://github.com/GenericMappingTools/egu22pygmt/issues/1 for more details.

Reply:
👍🏽